joint_trajectory_controller/src/joint_trajectory_controller.cpp

```diff
@@ -1,71 +1,95 @@
-///////////////////////////////////////////////////////////////////////////////
```

Comment:
I don't understand why git consider this code as removed and successively added.